README.md

```diff
@@ -169,7 +169,7 @@ We welcome contributions to this project! Here are a few guidelines to follow:
 
 ## References
 
-    1.  M. Glassman and M. J. Kang, “Intelligence in the internet age: The emergence and evolution of Open Source Intelligence (OSINT),” Comput. Human Behav., vol. 28, no. 2, pp. 673–682, 2012.
+    1.  M. Glassman and M. J. Kang, “Intelligence in the Internet age: The emergence and evolution of Open Source Intelligence (OSINT),” Comput. Human Behav., vol. 28, no. 2, pp. 673–682, 2012.
```

Comment:
Internet isn't capitalized in the book title. https://www.sciencedirect.com/science/article/abs/pii/S0747563211002585